### DISCUSSION TITLE
Help with DWIN Icon Tools

### DISCUSSION THREAD

**Discussion by embraceentropy** [Jyers/Marlin#868] [Q&A] [answered]
I'm trying to combine icons from the included custom & gotcha icons pack. I've followed the instructions in the included readme with no luck. I also tried to split an ICO file with no luck. I have python installed and I installed the images per instructions in that git. When I open the BAT a command window pops up but disappears before I can see what it is doing, then nothing happens.

**Comment by smooreace** [comment, marked as answer]
Same problem here. Wasn't important enough at the time to pursue a
resolution. But since it was brought up Ill toss in my towel as well

**Reply by tome9111991** [reply]
here a exemple how to make the 9.ico file
![make icon](https://user-images.githubusercontent.com/57866234/118999070-3faf2380-b98a-11eb-8870-76c8c9893f21.png)

**Reply by embraceentropy** [reply]
Thank you! I guess the BAT files are not working properly. I was able to create and explode ICO files using this method.

**Reply by Jyers** [reply]
The bat files should be working. Did you follow the instructions in the readme? The icon pack folder needs to end with "-Icons" and have no other spaces in the name

**Reply by embraceentropy** [reply]
@Jyers Yes, I followed the readme for makeico and splitico and tried several times with no success. Are others having success running them per the readme?

**Reply by Jyers** [reply]
It could also depend where the repo is on your computer, as the bat file might not have permission to write to some locations

**Reply by embraceentropy** [reply]
I tried moving the folder to root and desktop with the same result.

**Reply by Jyers** [reply]
Strange, I just tested both bat files on my machine and they worked fine. I wonder what the issue could be
